RecoLocalTracker/SiStripClusterizer/src/SiStripApvShotCleaner.cc

```diff
@@ -133,9 +133,10 @@ void SiStripApvShotCleaner::subtractCM(){
     return;
 
   //Subtract the median
+  const bool is10bit = apvDigis[0].adc() > 255; // approximation; definitely 10bit in this case
```

Comment:
Following your intentions explained in the PR description, I would expect here
```
const bool is10bit = apvDigis[0].adc() > 255;
for(size_t i=1; i<stripsForMedian;++i) is10bit |= apvDigis[i].adc() > 255;
```
Did I understand it correctly?

Reply:
Yes, but they are sorted by decreasing adc here (from https://github.com/cms-sw/cmssw/pull/24961/files#diff-af819317b5a71697a837cc86996841edL115 , to calculate the median) so the first is the largest (the implementation of this method could clearly be improved, but I wanted to keep the patch minimal).